File: isciProblemiCozenKod.java

import java.util.Scanner;

/**
 * Created by ahmetturkmen on 1/15/17.
 */
public class isciProblemiCozenKod {

    public static void main(String [] args){
        Scanner read= new Scanner(System.in);
        System.out.printf("Bir işçi işi kaç günde bitirmektedir? \n");
        int isiBitirmeSuresi= read.nextInt();
        System.out.printf("Toplam kaç işçi çalışacak? \n");
        int isciSayisi = read.nextInt();
        int isinBitmeSuresi = isiBitirmeSuresi/isciSayisi;
        System.out.printf("İşin bitme süresi %d gündür",isinBitmeSuresi);

    }
}

File: IsciProblemiCozenKod.py
# Bir işçinin bir işi kaç günde bitirdiği
gun = int(input("Bir işçi işi kaç günde bitirmektedir?: "))

# İşi bitirecek toplam işçi sayısı
isci = int(input("Toplam kaç işçi çalışacak?: "))

# İşin bitirme süresi
x = gun / isci

# Bölme işleminden dolayı sonucu noktalı sayıdan kurtarma
if x == int(x):
    x = int(x)
else:
    x = round(x, 3)

print("İşin bitirme süresi:", x, "gündür.")
